src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+.body {
+  max-width: 1200px;
```

Comment:
Adding max-width directly to the body element is generally not considered a best practice because it can have unintended consequences and limit your flexibility in styling and layout.